authors/apps/authentication/backends.py

```diff
@@ -1,15 +1,80 @@
-# import jwt
-#
-# from django.conf import settings
-#
+import jwt
+from django.conf import settings
 from rest_framework import authentication, exceptions
-#
-# from .models import User
+from .models import User
 
 """Configure JWT Here"""
 
 
 class JWTAuthentication(authentication.BaseAuthentication):
+    """This class handles authentication of the JSON Web Tokens provided by the user"""
 
     def authenticate(self, request):
-        pass
+        """This method checks the Authorization header of every request
+
+        Args:
+            param1: request.
```

Comment:
@calvinpete update args e.g
```suggestion
           request (request obj): django request context
```

Reply:
Noted.